ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -383,3 +383,56 @@ periodics:
     - configMap:
         name: prow-job-cluster-launch-e2e-azure-conformance
       name: job-definition
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: periodic-ci-azure-bushslicer
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --git-ref=openshift/openshift-azure@master
+      - --secret-dir=/usr/local/e2e-secrets/azure
+      - --target=e2e-azure
+      - --template=/usr/local/e2e-azure
+      command:
+      - ci-operator
+      env:
+      - name: CLUSTER_TYPE
+        value: azure
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: openshift-openshift-azure-master.yaml
+            name: ci-operator-configs
+      - name: JOB_NAME_SAFE
+        value: e2e-azure
+      - name: TEST_IMAGE
+        value: quay.io/kwoodson/bushslicer
+      - name: TEST_COMMAND
+        value: "/usr/bin/scl enable rh-git29 rh-ror50 -- cucumber features/cli/build.feature:5"
```

Comment:
In the near future, we'll want to update this with the full list of testcases to run. For now, this is fine as a place holder. Thanks!

Reply:
@cjryan, please supply the full list.  I think it's time.